guide/principles/principles.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: guide
+title: Principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 9
+parent: Foundations
+permalink: /guide/foundations/principles/
+main_classes: -no-top-padding
+image: /assets/images/guide/foundations/principles/page-principles.jpg
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/foundations/principles/principles.jpg"
+   retina = "/assets/images/guide/foundations/principles/principles@2x.jpg"
+   mobile = "/assets/images/guide/foundations/principles/principles-mobile.jpg"
+   mobileRetina = "/assets/images/foundations/principles/principles-mobile@2x.jpg"
+   alt-text = "Principles header illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system, where participants share the role of securing the network. This is important to give everyone equal and direct access without needing intermediaries. To make this a reality, we encourage everyone working on products to deliberately support the core principles of designing for Bitcoin. 
```

Comment:
Is it important to say why not having intermediaries is something to strive for (like unfairly excluding certain people from being economically active, unfairly messing with peoples savings, etc)?

Reply:
Expanded.